task_12/src/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$font: 'Roboto';
```

Comment:
Actually, when you somewhere use `font-family: $font;`, you do not know is it `serif` or `sans-serif`. You need to pass whole font list here and in the main use only variable.

Also, add fall-back fonts you prefer. When the list is `Roboto, sans-serif` and Roboto failed to load, on Windows you get Arial and on MacOS you get Helvetica. It's okay but if you prefer other fonts add it to the list.

To see current default sans-serif fonts for each system [follow the link](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L230).